### PR TITLE
Bump nodejs-compression-webpack-plugin to 1.1.11

### PR DESCRIPTION
**Pull request by mmoll**
this should be built into:

* [x] Nightly

### CODE REVIEW

**Comment by ekohl** [comment]
[test rpm]

File: packages/foreman/nodejs-compression-webpack-plugin/nodejs-compression-webpack-plugin.spec [hunk 1]
```diff
@@ -1,62 +1,201 @@
...
 %files
 %{nodejs_sitelib}/%{npm_name}
-
-%doc README.md
+%license node_modules/%{npm_name}/LICENSE
+%doc node_modules/%{npm_name}/CHANGELOG.md
+%doc node_modules/%{npm_name}/README.md
 
 %changelog
+* Sat Jun 16 2018  1.1.11-1
```
Comment (ekohl) [inline packages/foreman/nodejs-compression-webpack-plugin/nodejs-compression-webpack-plugin.spec]: The author is missing here

Comment (mmoll) [reply packages/foreman/nodejs-compression-webpack-plugin/nodejs-compression-webpack-plugin.spec]: updated.

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**